kuzzleio/documentation
# Add postgres how-to to the list

## What does this PR do?
This PR adds the [Postgres replication](https://github.com/kuzzleio/kuzzle-how-to/tree/master/replicate-to-sql-with-generic-events) to the list of available How-to

Add logo & links in the how-to.json

### How should this be manually tested?

1. clone the repo
1. npm run doc-dev
1. localhost:8080 -> click "How to"
1. See that "Replicate with Postgres" appear
